_data/tool_and_resource_list.yml

```
@@ -215,6 +215,10 @@
   id: testthat
   name: testthat
   url: https://testthat.r-lib.org/
+- description: Test module for Julia
+  id: "test.jl"
```

```suggestion
  id: "Test.jl"
```
Id must be the same as the one referenced in text

---

And it's better with the correct capitalisation! Thanks @dgarijo.